sdk/packages/eternum/src/config/index.ts

```diff
@@ -219,8 +219,14 @@ export const setupGlobals = async (account: Account, provider: EternumProvider)
 
   const txExplore = await provider.set_exploration_config({
     signer: account,
-    wheat_burn_amount: EternumGlobalConfig.exploration.wheatBurn * EternumGlobalConfig.resources.resourcePrecision,
-    fish_burn_amount: EternumGlobalConfig.exploration.fishBurn * EternumGlobalConfig.resources.resourcePrecision,
+    explore_wheat_burn_amount:
```

Comment:
add resourceMultiplier constant as well, so that it's following the same pattern as the rest

Reply:
not sure what you want me to do here. but the app works as it intended without modifying this

Reply:
what i meant is that in the config there's 2 constant resource precision and multiplier:
<img width="265" alt="image" src="https://github.com/user-attachments/assets/8857a45a-4d2a-4af4-aeb6-3d8b22040ca2">

resource multiplier means that you can arbitrarily change the amount of resources, and it will be changed in the whole app. So if all of a sudden, we want resources to be produced 10x less, we set the mutliplier to 100 instead of 1000.

Reply:
you can see resourceMultiplier being used multiple places in the code

Reply:
nvm lgtm